webapi/scripts/expected-client-test-data.txt

```diff
@@ -14,6 +14,8 @@ test-data/admin/groups/get-groups-response.json
 test-data/admin/groups/update-group-request.json
 test-data/admin/groups/update-group-response.json
 test-data/admin/lists/
+test-data/admin/lists/add-child-to-second-child-request.json
+test-data/admin/lists/add-child-to-second-child-response.json
```

Comment:
why has this changed in this PR?

Reply:
As far as I see it is just filename adjustment to match the test name.